### PR TITLE
Fix airlock lighting

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Fixes airlock lighting. Name your args!

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/42d17f7c-2da0-4a8c-b8c9-3775bb69b3a7

</details>

## Changelog

:cl: LT3
fix: Airlock lighting works again
/:cl:

### CODE REVIEW

**Review by softcerv — COMMENTED** [review]
Yeah, this looks fine.

File: modular_skyrat/modules/aesthetics/airlock/code/airlock.dm [hunk 1]
```diff
@@ -131,9 +131,9 @@
 		pre_light_range = door_light_range
 		pre_light_power = door_light_power
 		if(has_environment_lights)
-			set_light(pre_light_range, pre_light_power, pre_light_color, TRUE)
+			set_light(l_range = pre_light_range, l_power = pre_light_power, l_color = pre_light_color, l_on = TRUE)
 			if(multi_tile)
-				filler.set_light(pre_light_range, pre_light_power, pre_light_color)
+				filler.set_light(l_range = pre_light_range, l_power = pre_light_power, l_color = pre_light_color, l_on = TRUE)
```
Comment (vinylspiders) [inline modular_skyrat/modules/aesthetics/airlock/code/airlock.dm:136]:
```suggestion
				filler.set_light(l_range = pre_light_range, l_power = pre_light_power, l_color = pre_light_color, l_on = TRUE)
		else
			set_light(0)
```

Comment (vinylspiders) [reply modular_skyrat/modules/aesthetics/airlock/code/airlock.dm:136]: I don't see the light being turned off anywhere, shouldn't we add this too?

**Review by vinylspiders — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/aesthetics/airlock/code/airlock.dm [hunk 2]
```diff
@@ -131,9 +131,9 @@
...
 	else
 		lights_overlay = ""
```
Comment (vinylspiders) [inline modular_skyrat/modules/aesthetics/airlock/code/airlock.dm:138]:
```suggestion
		set_light(0)
		lights_overlay = ""
```

Comment (vinylspiders) [reply modular_skyrat/modules/aesthetics/airlock/code/airlock.dm:138]: Same thing here

File: modular_skyrat/modules/aesthetics/airlock/code/airlock.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply modular_skyrat/modules/aesthetics/airlock/code/airlock.dm:136]: I'd think so yeah.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by vinylspiders — APPROVED** [review]

**Review by GoldenAlpharex — APPROVED** [review]
Amazing work, thank you!

**Pull request merged**